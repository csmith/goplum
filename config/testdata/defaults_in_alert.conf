# Test that defaults blocks cannot be nested inside alert blocks
alert smtp.send "email-alerts" {
  server = "mail.example.com:25"
  from = "alerts@example.com"
  to = "admin@example.com"
  
  # This should not be allowed
  defaults {
    subject_prefix = "ALERT: "
  }
}