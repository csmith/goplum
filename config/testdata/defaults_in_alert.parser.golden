"unexpected defaults keyword (defaults) at line 6 column 33, expecting one of: end of block, identifier"